Podfile

```diff
@@ -4,6 +4,7 @@ use_frameworks!
 def common_pods
   pod 'SwiftLint', '0.27.0'
   pod 'SQLite.swift', '0.11.5'
+  pod 'BTNavigationDropdownMenu'
```

Comment:
Should the version be specified in the podfile? It might be a good idea for clarity
The line would look like `pod 'BTNavigationDropdownMenu', '0.5'

might not be necessary, just a thought